demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1128,7 +1128,10 @@ export class DemoMeetingApp
       enableWebAudio: this.enableWebAudio,
     });
     configuration.enableUnifiedPlanForChromiumBasedBrowsers = this.enableUnifiedPlanForChromiumBasedBrowsers;
-    configuration.attendeePresenceTimeoutMs = 5000;
+    const urlParameters = new URL(window.location.href).searchParams;
+    if (urlParameters.has('attendee-presence-timeout-ms')) {
```

Comment:
Use `isNaN` to check if the query string can be converted to a string.

```
const timeoutMs = urlParameters.get('attendee-presence-timeout-ms');
if (!isNaN(timeoutMs)) {
  configuration.attendeePresenceTimeoutMs = Number(timeoutMs);
}
```

Reply:
Removed the check urlParameters.has() and used isNaN instead.